packages/password-input/src/PasswordInput/PasswordInput.types.ts

```diff
@@ -0,0 +1,117 @@
+import { ChangeEventHandler, ComponentPropsWithoutRef } from 'react';
+
+import { DarkModeProps, Either } from '@leafygreen-ui/lib';
+
+export const States = {
```

Comment:
nit: the name of this enum/type should be singular (e.g. `Size`, `Theme`, `Variant` etc)

It's a bit confusing to see something like `state: States = ...`. Feels like it should be an array

Reply:
oh you're right, i'll update this